### ISSUE TITLE
CREATE TABLE DDL not transformed properly

### ISSUE THREAD

**Issue by subkanthi** [Altinity/clickhouse-sink-connector#321]
```
CREATE TABLE `t1` (
  `id` bigint NOT NULL AUTO_INCREMENT,
  `last_update` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
  `h1` varchar(64) CHARACTER SET ascii COLLATE ascii_general_ci NOT NULL,
  `v1` varchar(4096) CHARACTER SET ascii COLLATE ascii_general_ci NOT NULL,
  PRIMARY KEY (`id`),
  UNIQUE KEY `hint_uidx` (`h1`)
) ENGINE=InnoDB AUTO_INCREMENT=157407 DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs
```

**Comment by subkanthi** [comment]
In `23.3.4.17`, the table was created in Clickhouse
```
show create table t1;

SHOW CREATE TABLE t1

Query id: d40e1c9c-75fe-48eb-a5b0-3cb3a41494b6

┌─statement──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│ CREATE TABLE test.t1
(
    `id` Int64,
    `last_update` DateTime64(3),
    `h1` String,
    `v1` String,
    `_version` UInt64,
    `is_deleted` UInt8
)
ENGINE = ReplacingMergeTree(_version, is_deleted)
ORDER BY id
SETTINGS index_granularity = 8192 │
└────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘

1 row in set. Elapsed: 0.002 sec. 

a96470a11fe1 :) select version();

SELECT version()

Query id: 31c3c353-9fcb-43b6-b4ca-cfa4a6fea2e7

┌─version()─┐
│ 23.3.4.17 │
└───────────┘

1 row in set. Elapsed: 0.002 sec. 
```

**Comment by subkanthi** [comment]
seems to work even with `allow_nullable_key`

```
CREATE TABLE test.t5
(
    `id` Int64,
    `last_update` DateTime64(3),
    `h1` String,
    `v1` String,
    `_version` UInt64,
    `is_deleted` UInt8
)
ENGINE = ReplacingMergeTree(_version, is_deleted)
ORDER BY id
SETTINGS index_granularity = 8192, allow_nullable_key = 1
```

**Comment by subkanthi** [comment]
Adding `not null` similar to the python version.

**Comment by subkanthi** [comment]
Fixed in #356
